pylint/checkers/utils.py

```diff
@@ -1686,3 +1687,16 @@ def returns_bool(node: nodes.NodeNG) -> bool:
         and isinstance(node.value, nodes.Const)
         and node.value.value in (True, False)
     )
+
+
+T_Node = TypeVar("T_Node", bound=nodes.NodeNG)
```

Comment:
Could you move this to the top of the file? We tend to keep our `TypeVar`'s there.